airbyte-integrations/connectors/source-merge/source_merge/manifest.yaml

```diff
@@ -0,0 +1,183 @@
+version: "0.29.0"
+
+definitions:
+  selector:
+    type: RecordSelector
+    extractor:
+      type: DpathExtractor
+      field_path: ["results"]
+
+  requester:
+    type: HttpRequester
+    url_base: "https://api.merge.dev/api/ats/v1"
+    http_method: "GET"
+    request_headers:
+      X-Account-Token: "{{ config['account_token'] }}"
+    authenticator:
+      type: BearerAuthenticator
+      api_token: "{{ config['api_token'] }}"
+
+  selective_paginator:
+    type: DefaultPaginator
+    pagination_strategy:
+      type: "CursorPagination"
+      cursor_value: "{{ response.next }}"
+    page_token_option:
+      type: RequestPath
+
+  retriever:
+    type: SimpleRetriever
+    record_selector:
+      $ref: "#/definitions/selector"
+    paginator:
+      $ref: "#/definitions/selective_paginator"
+    requester:
+      $ref: "#/definitions/requester"
+
+  base_stream:
+    type: DeclarativeStream
+    retriever:
+      $ref: "#/definitions/retriever"
+  
+  account_details_stream:
```

Comment:
Why you're not adding the `/accounts` endpoint?

Reply:
The /accounts is for Accounting Base, the issue was inlt based for ATS URL
See https://docs.merge.dev/ats/overview/

All the endpoints inside Merge ATS is added :)